pyais/messages.py

```diff
@@ -155,6 +155,51 @@ def wrapper(tb: 'TagBlock') -> typing.Any:
     return wrapper
 
 
+class TagBlockGroup:
+    """Tag Block Group represents the 3-int group sequence 
+    optionally included as part of the NMEA Tag Block
+
+    it consists of 3, comma-seperated integers X-Y-Z where:
+    - X = Message ID in sequence
+    - Y = Total parts in group
+    - Z = Unique GroupID for this group of messages."""
+
+    __slots__ = (
+        'msg_id',
```

Comment:
I would prefer to name this attribute `sentence_number` or `sentence_num`

Reply:
Of course. I'm very amenable on var names. Please comment any other suggestions you may have as well and I'll update them all in batch! 😇

Reply:
Let's name them `sentence_num` and `sentence_tot`